### PR TITLE
cf919d add c++ and python solutions

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by 876pol** [comment]
the python code ACs on pypy 64 bit (https://codeforces.com/contest/919/submission/190263892) but not regular pypy (https://codeforces.com/contest/919/submission/190263898). should i mention something about this on the editorial?

File: solutions/gold/cf-919D.mdx [hunk 1]
```diff
@@ -26,6 +26,80 @@ represents $s_i$, since we can pick up one more of this letter on our path.
...
+
+	// find the maximum frequency across all nodes and letters.
+	int ans = 0;
+	for (int i = 0; i < n; i++) {
+		for (int j = 0; j < 26; j++) {
+			ans = max(ans, dp[i][j]);
+		}
+	}
+	cout << ans << endl;
+	return 0;
```
Comment (SansPapyrus683) [inline solutions/gold/cf-919D.mdx:98]:
```suggestion
```

File: solutions/gold/cf-919D.mdx [hunk 2]
```diff
@@ -105,4 +179,59 @@ public class Substring {
...
+		in_degree[nxt] -= 1
+		if in_degree[nxt] == 0:
+			queue.append(nxt)
+	size += 1
+
+# no answer.
+if size < n:
+	print(-1)
+# find the maximum frequency across all nodes and letters.
+else:
```
Comment (SansPapyrus683) [inline solutions/gold/cf-919D.mdx:228]:
```suggestion
else:  # find the maximum frequency across all nodes and letters.
```
also if you're gonna use periods might as well go all the way & capitalize sentences asw

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by SansPapyrus683** [comment]
> the python code ACs on pypy 64 bit (https://codeforces.com/contest/919/submission/190263892) but not regular pypy (https://codeforces.com/contest/919/submission/190263898). should i mention something about this on the editorial?

yes

**Review by SansPapyrus683 — APPROVED** [review]
seems legit

**Review by dongliu0426 — APPROVED** [review]

**Pull request merged**